8Knot/assets/color.css

```diff
@@ -32,25 +56,32 @@ blockquote p {
   max-width: 950px;
 }
 
-/* this is to change the color for the spinner, cant be changed directly */
+/* ==== DASH SPECIFIC COMPONENTS ==== */
+/* Spinner needs !important because it's deeply nested in Dash internals */
```

Comment:
is this comment needed now that !important has been removed?

Reply:
Is this spinner suppose to be for when the graph loads? if so that color needs to be updated to the new styling